environment_python_3_7_with_bokeh2.1.1.yml

```diff
@@ -0,0 +1,40 @@
+channels:
```

Comment:
@bhilbert4 did you mean to add this file?

Reply:
Removed. Not sure how it got in there.